node-backend/app/models/sequelize.js

```diff
@@ -1,6 +1,7 @@
 const Sequelize = require("sequelize");
 
-const dbConfig = require("../config/db.config.js");
+const dbConfigFunction = require("../config/db.config.js");
```

Comment:
You should add 
const env = process.env.NODE_ENV || 'development';
const config = require(../config/db.config.js)[env];
you will get env config by this way